spec/fluent/plugin/prometheus_metrics_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+require 'fluent/plugin/in_prometheus'
+require 'fluent/test/driver/input'
+
+require 'net/http'
+
+describe Fluent::Plugin::PromMetricsAggregator do
+
+  metrics_worker_1 = %[# TYPE fluentd_status_buffer_queue_length gauge
```

Comment:
Please add summary type for the test

Reply:
I added histogram metrics to the tests in 99dfdff8b783c7dd8d4e550100e1e9a8bcb3ffac.
The metrics merge logic also works with summaries but I'm not sure this metric type is useful in a multi worker setup (aggregation of precomputed quantiles from multiple workers doesn't make sense to me).